### PR TITLE
Adding a file about Mountain View

### LINKED ISSUE #16: Create a new file for tamarasavage2 hometown
## Add file for Mountain View, CA
- [x] Create a branch
- [x] Add my file
- [x] Create a commit
- [x] Open a pull request
- [ ] Have a discussion about the pull request
- [ ] Merge our pull request

### PR DESCRIPTION
**Pull request by tamarasavage2**
This is my hometown @tamarasavage2 .

This Pull request closes #16

### CODE REVIEW

File: places/mountain-view.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Mountain View, CA
+
+### Great places to eat
```
Comment (tamarasavage2) [inline places/mountain-view.md:3]: @tamarasavage2 I :heart: Italian - do you have any recs?

Comment (tamarasavage2) [inline places/mountain-view.md:3]: Just having a conversation with myself here

**Comment by Rosanne6** [comment]
Ship it!!!  --rosanne

**Comment by jdkjoel** [comment]
looks good to me.  :shipit:

**Comment by nguyengithub** [comment]
your code looks good.  Thanks

**Pull request merged**